src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -362,7 +363,7 @@ export class Kernel implements IKernel {
             return chainWithPendingUpdates(notebookDocument, (edit) => {
                 const markdownCell = new NotebookCellData(
                     NotebookCellKind.Markup,
-                    sysInfoMessages.join('\n\n'),
+                    sysInfoMessages.join('  \n'),
```

Comment:
To split markdown into multiple lines we need to ensure the markdown line ends with a trailing 2 empty spaces (standard in markdown, hence not adding comments to that regard).

Reply:
This way we don't end up with empty lines in markdown content.